charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -63,5 +63,8 @@ alfresco-sync-service:
 
 alfresco-infrastructure:
   enabled: true
+  #DEPLOY-613 - Multiple ActiveMQ instances are deployed
```

Comment:
Can we add a TODO to remove as part of DEPLOY-563?

Reply:
Added.